nexus_constructor/stream_fields_widget.py

```diff
@@ -53,18 +53,18 @@
 ADC_PULSE_DEBUG = "adc_pulse_debug"
 
 
-def check_if_advanced_options_should_be_enabled(
-    elements: List[str], field: h5py.Group
-) -> bool:
+def check_if_advanced_options_should_be_enabled(elements: List[str], field) -> bool:
     """
     Checks whether the advanced options box should be enabled by checking if any of the advanced options have existing values.
     :param elements: list of names to check if exist
     :param field: the field group
     """
-    return any(item in field.keys() for item in elements)
+    return False
```

Comment:
Please add the same searchable comment you've added elsewhere so that this doesn't get left here by accident.